verl/utils/fsdp_utils.py

```diff
@@ -57,8 +59,24 @@ def get_fsdp_wrap_policy(module, config=None):
                                                     default_transformer_cls_names_to_wrap)
     min_num_params = config.get('min_num_params', 0)
     auto_wrap_policy = None
+    
+    policies = []
+    
+    from torch.distributed.fsdp.wrap import _or_policy, lambda_auto_wrap_policy, transformer_auto_wrap_policy
```

Comment:
I guess we should only turn on this policy when we use lora. Otherwise, the performance may degrade

Reply:
done!